src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,70 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+
+import cn from 'classnames';
+
+import { Tab } from '../../types';
+
+type Props = {
+  onTabSelected: (tabNr: Tab) => void,
+  tabs: Tab[] | [],
+  selectedTabId: string,
+};
+
+export const Tabs: FC<Props> = ({
+  onTabSelected,
+  tabs,
+  selectedTabId,
+}) => {
+  const tabContent = tabs.find(tab => (
+    tab.id === selectedTabId
+  ))?.content;
+
+  const isIdCorrect:boolean = tabs.some((tab:Tab) => (
+    tab.id === selectedTabId
+  ));
+
+  return (
+    <div className="TabsComponent" data-cy="tab-content">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map((tab, index) => {
+              const { id, title } = tab;
```

Comment:
```suggestion
            tabs.map(({ id, title }, index) => {
```

Reply:
@jstmpelowycz I don't think I can use this destructuring because I am passing tab object into onTabSelected below